.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: Release
```

Comment:
please remove this workflow. this is also a monorepo and needs a manual way to do releases.

Reply:
fixed